CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v1.4.2 (unreleased)
```

Comment:
I just use the "Next release" header for unreleased changes as the version number won't be known until preparing the release and depending upon what it contains (breaking changes or minor bugfixes).